cmsplugin_filer_file/models.py

```diff
@@ -37,8 +37,8 @@ def has_attached_file(self):
     def get_icon_url(self):
         return self.file.icons['32'] if self.has_attached_file() else None
 
-    def get_adjusted_icon_url(self):
-        return self.get_icon_url().replace(settings.STATIC_URL, "", 1)
+    def is_absolute_url(self):
+        return bool(urlparse(self.get_icon_url()).netloc)
```

Comment:
Since get_icon_url can return None it should be better to use self.get_icon_url() or ''